general/clients/css-customization.md

```diff
@@ -69,18 +69,28 @@ This will affect the played/watched indicator. Replace the color hex with any va
 
 ![Screenshot of watched indicators with a custom transparent color applied](~/images/custom-css-transparentwatched.png)
 
-#### Remove Live TV Channel Listings
+### Remove Live TV Channel Listings
 
 ```css
 .guideChannelNumber { display: none; }
 ```
 
-#### Reduce Live TV Channel Width
+### Reduce Live TV Channel Width
 
 ```css
 .channelsContainer { max-width: 8em; }
 ```
 
+### Background Image on Homepage
+
+[Additional MDN Documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/background)
+
+```css
+#indexPage {
+    background: url(https://i.ytimg.com/vi/avCWDDox1nE/maxresdefault.jpg);
```

Comment:
Heh it's official now :P
Should we add `background-size: cover` if we're officialising this?

Reply:
Might be a good idea to include the background-size too. But in that case `background` should be `background-image`

Reply:
updated